Sources/Application/View/MockView.swift

```diff
@@ -14,6 +19,19 @@ public class MockView: Viewable {
 		displayLosingMessageCalled = false
 	}
 
+	public func promptNumberOfGuesses() {
+		self.promptNumberOfGuessesCalled = true
+	}
+
+	public func receiveNumberOfGuesses() -> Int {
+		receiveNumberOfGuessesCalled = true
+		return receiveNumberOfGuessesReturn
```

Comment:
@mnewsome this return is necessary. I realized I was not using it. I need a return value because the regular View and protocol have a return value. Specs have been updated to utilize this and the one for guess